doc/manual/utils.nix

```
@@ -38,4 +38,46 @@ rec {
 
   filterAttrs = pred: set:
     listToAttrs (concatMap (name: let v = set.${name}; in if pred name v then [(nameValuePair name v)] else []) (attrNames set));
+
+  showSetting = useAnchors: name: { description, documentDefault, defaultValue, aliases, ... }:
```

I don't see why this was moved into `utils.nix`, since `generate-options.nix` was a suitable single source of truth. Possibly `option -> setting` may be appropriate. Utils seem to be a barebones version of `pkgs.lib`, but nothing specific to documentation.

---

```suggestion
  showSetting = useAnchors: name: { description, documentDefault, defaultValue, aliases }:
```

Also we probably want this to break when we add more fields, so it prompts us to accommodate them, right?